LamestWebserver/LamestWebserver/Core/Logger.cs

```diff
@@ -24,11 +26,68 @@ public class Logger
         public static ELoggingLevel DefaultMinimumLoggingLevel = ELoggingLevel.Trace;
 
         /// <summary>
+        /// Path for the Logging File
+        /// </summary>
+        public static string FilePath
+        {
+            get
+            {
+                return CurrentLogger.Instance._currentFilePath;
+            }
+            set
+            {
+                CurrentLogger.Instance._currentFilePath = value;
+                CurrentLogger.Instance.Close();
+                CurrentLogger.Instance.Open();
+            }
+        }
+
+        /// <summary>
         /// The minimum logging level for this logger.
         /// </summary>
         public ELoggingLevel MinimumLoggingLevel = DefaultMinimumLoggingLevel;
 
         /// <summary>
+        /// Set the Flags for the Output Sources
+        /// </summary>
+        public EOutputSource OutputSourceFlags
+        {
+            get
+            {
+                return _currentOutputSource;
+            }
+            set
+            {
+                _currentOutputSource = value;
+                CurrentLogger.Instance.Close();
+                CurrentLogger.Instance.Open();
+            }
+        }
+
+        /// <summary>
+        /// Stream Writer to handle the writing from multiple streams
+        /// </summary>
+        private MultiStreamWriter _multiStreamWriter;
+
+        /// <summary>
+        /// Currently used File Path
+        /// </summary>
+        private string _currentFilePath = "LWS" + DateTime.Now.ToFileTime().ToString() + ".log";
```

Comment:
There should also be a way to set this for other Loggers than the Default Logger.
Remember, that Logger.Instance is __not__ a singleton, but the default instance.
Maybe make this just public?